src/source/Rtp/RtpPacket.c

```diff
@@ -168,45 +169,59 @@ STATUS setRtpPacketFromBytes(PBYTE rawPacket, UINT32 packetLength, PRtpPacket pR
     BOOL marker;
     UINT8 payloadType;
     UINT16 sequenceNumber;
+    UINT16 i;
     UINT32 timestamp;
     UINT32 ssrc;
     PUINT32 csrcArray = NULL;
+    PUINT32 extensionPayloadWord = NULL;
     UINT16 extensionProfile = 0;
     UINT16 extensionLength = 0;
     PBYTE extensionPayload = NULL;
-    UINT32 currOffset = 0;
-
-    CHK(pRtpPacket != NULL && rawPacket != NULL, STATUS_NULL_ARG);
+    UINT32 word;
```

Comment:
I know this is a bit of knit picking. I think it will be good if we initialize all or none. Here some of the variables are initialized and some are not. Please do it in the other functions as well.

Reply:
I think webrtc SDK is not following the rule to initialising all the variables, hence only the variables whose values are required was initialised and others are initialised before use (the new variables). Hence i will keep it as it is for now, as i do not want to update existing variables.